modules/picard/sortvcf/main.nf

```diff
@@ -0,0 +1,45 @@
+process PICARD_SORTVCF {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::picard=2.26.10" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:2.26.10--hdfd78af_0' :
+        'quay.io/biocontainers/picard:2.26.10--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(vcf)
+    path sequence_dict
+
+    output:
+    tuple val(meta), path("*_sorted.vcf"), emit: vcf
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
```

Comment:
Does `picard Sortvcf` take `$args`?

Reply:
Good suggestion! Will fix it 👍🏻

Reply:
Turns out `--CREATE_MD5_FILE <Boolean>`  only works for bam and fastq files, and not vcfs. But I have added an optional input for `--REFERENCE_SEQUENCE` now.